### PR TITLE
Batch execution blog

### PR DESCRIPTION
**Pull request by dawidwys**

### CODE REVIEW

**Comment by dawidwys** [comment]
cc @morsapaes

File: _posts/2021-03-10-batch-execution-mode.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title:  "A Rundown of Batch Execution Mode in the DataStream API"
+date: 2021-03-10T00:00:00.000Z
+categories: news
```
Comment (morsapaes) [inline _posts/2021-03-10-batch-execution-mode.md]:
```suggestion
```

Comment (morsapaes) [reply _posts/2021-03-10-batch-execution-mode.md]: This is reserved for release announcements and such. 😉

**Comment by morsapaes** [comment]
Thanks, @dawidwys ! Can you just please remove all the files that are not .md or images? The website is rebuilt by the committer when merging.

**Comment by rmetzger** [comment]
@dawidwys do you want to merge this PR?

**Comment by dawidwys** [comment]
If it's good, sure I can.

**Pull request closed without merging**